packages/twenty-front/src/modules/workflow/types/Workflow.ts

```diff
@@ -62,7 +60,28 @@ export type WorkflowDatabaseEventTrigger = BaseTrigger & {
   };
 };
 
-export type WorkflowTrigger = WorkflowDatabaseEventTrigger;
+export type WorkflowManualTrigger = BaseTrigger & {
+  type: 'MANUAL';
+  settings:
+    | {
+        availability: 'EVERYWHERE';
+      }
+    | {
+        availability: 'WHEN_RECORD_SELECTED';
+        objectType: string;
+      };
```

Comment:
Makes sense to me too 👍

Reply:
I don't see the benefit of going with a generic `input` object. Please correct me.

Here, I'm setting some configurations for the manual trigger. Currently, a manual trigger is either available on every page (because it doesn't depend on any record), or on the pages related to a specific record type. That's hard configuration, not something that should change from one manual trigger to another one.

"Availability" is the wording we chose in the frontend because the manual trigger will be available either _everywhere_ or _when record x is selected_.

Do I miss something? 🤔